alerts.js

```diff
@@ -0,0 +1,182 @@
+require('dotenv').config()
+const BN = require('bignumber.js')
+const Web3 = require('web3')
+const logger = require('./logger')('alerts')
+const eventsInfo = require('./utils/events')
+
+const { HOME_RPC_URL, FOREIGN_RPC_URL } = process.env
+
+const homeProvider = new Web3.providers.HttpProvider(HOME_RPC_URL)
+const web3Home = new Web3(homeProvider)
+
+const foreignProvider = new Web3.providers.HttpProvider(FOREIGN_RPC_URL)
+const web3Foreign = new Web3(foreignProvider)
+
+async function main() {
+  try {
+    const { foreignDeposits, homeDeposits, homeWithdrawals, foreignWithdrawals } = await eventsInfo()
+
+    const xSignatures = foreignDeposits.filter(findDifferences(homeDeposits))
+    const xAffirmations = homeWithdrawals.filter(findDifferences(foreignWithdrawals))
+
+    logger.debug("building misbehavior blocks")
+    const getBlockNumber = web3 => web3.eth.getBlockNumber()
+    const toBN = n => new BN(n)
```

Comment:
Do you think it would be a good idea to use `Web3.utils.toBN` instead of defining a new `toBN` method?

Reply:
Yes, completely forgot about the `Web3`'s `BN` library.

Just changed. Thanks!